src/sentry/features/manager.py

```diff
@@ -124,7 +124,7 @@ def _get_feature_class(self, name):
         except KeyError:
             raise FeatureNotRegistered(name)
 
-    def get(self, name, *args, **kwargs):
+    def get(self, name: str, *args, **kwargs):
```

Comment:
No typing for `*args` and `**kwargs`?

Reply:
I don't think you can reasonably type this except for some common args.

Reply:
I don't think that's true. If the variadic arguments are all supposed to be `int`s then that should be noted. Even adding `Any` tells me that I can't make any assumptions about what these values are.